pkg/sdk/system_functions.go

```diff
@@ -26,3 +27,18 @@ func (c *systemFunctions) GetTag(ctx context.Context, tagID ObjectIdentifier, ob
 	}
 	return s.Tag, nil
 }
+
+func (c *systemFunctions) GetTags(ctx context.Context, tagID ObjectIdentifier, objectID ObjectIdentifier, objectType ObjectType) ([]string, error) {
+	res := []struct {
+		Tag string `db:"TAG"`
+	}{}
+	sql := fmt.Sprintf(`SELECT SYSTEM$GET_TAG('%s', '%s', '%s') TAG_VALUE WHERE TAG_VALUE IS NOT NULL`, tagID.FullyQualifiedName(), objectID.FullyQualifiedName(), objectType)
```

Comment:
Does this method even work? `Tag string db:"TAG"` does not match `TAG_VALUE`. And also, how does it differ from the one above? Is it possible to set multiple and check them like this?

Reply:
copied the logic as it was in the Snowflake package. i wasn't sure if the len of tags was necessary for retry logic, such as in the case where the tag association hasn't been created, but no error. I tried using the GetTag by itself, and it seems to work fine. so I have removed the GetTags code